_spinnaker/terraform_integration.md

```diff
@@ -142,64 +141,71 @@ hal armory terraform edit \
 
 ### Selecting the Terraform version
 
-Terraformer currently ships with the following versions of the Terraform binary:
+Armory ships the following versions of the Terraform binary as part of the Terraform Integration:
 
 * 0.11.10 through 0.11.14
 * 0.12.0 through 0.12.10
```

Comment:
This is up to 0.12.18 now. Also seems awkward to manually update this all the time

Reply:
+1, agreed. We should figure out a way to let users get any version they want as long as it's available from the download page.